tool/src/java/org/sakaiproject/gradebookng/tool/panels/AssignmentColumnHeaderPanel.java

```diff
@@ -75,7 +75,7 @@ public void onInitialize() {
 			} else if ("Lesson Builder".equals(assignment.getExternalAppName())) {
 				iconClass = "icon-sakai-lessonbuildertool";
 			}
-			externalAppFlag.add(new AttributeModifier("class", "gb-external-app-flag " + iconClass));
+			externalAppFlag.add(new AttributeModifier("class", "gb-external-app-flag Mrphs-toolsNav__menuitem--icon icon-active " + iconClass));
```

Comment:
I know this isn't yours but what is up with that crazy class name? Single hyphen, double hyphen, double underscore, spaces, oh my! Is it generated?